packages/peregrine/lib/talons/Cms/useCmsPage.js

```diff
@@ -60,7 +61,6 @@ export const useCmsPage = props => {
     return {
         cmsPage,
         hasContent,
-        error,
```

Comment:
I think I'd prefer to keep returning this value from the error, even if we don't use it in the component.

Reply:
Reverted